wordpress-mobile/WordPress-Android
# [Blogging prompts] Prompts list item opens editor with prompt block

Fixes #17734

To test:
1 - Install Jetpack and log in;
2- Open "Debug settings" and make sure `BloggingPromptsEnhancementsFeatureConfig` is disabled;
3 - Open "My Site" -> "HOME" with a blog selected;
4 - Open the overflow menu in the prompt card and tap on "View more prompts" to open the prompts list;
5 - Tap on any list item: editor **should not** be opened. The analytics event `BLOGGING_PROMPTS_LIST_ITEM_TAPPED` should appear as tracked on the logcat;
6 - Open "Debug settings" and enable BloggingPromptsEnhancementsFeatureConfig;
7 - Open prompts list again;
8 - Tap on any list item: the Editor **should** be opened with the prompt block. The analytics event `BLOGGING_PROMPTS_LIST_ITEM_TAPPED` should appear as tracked on the logcat.

## Regression Notes
1. Potential unintended areas of impact
None.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing and updated unit tests.

3. What automated tests I added (or what prevented me from doing so)
Update `BloggingPromptsListAnalyticsTrackerTest.kt` and `BloggingPromptsListViewModelTest.kt`.

PR submission checklist:
